bundles/org.openhab.binding.meteoalerte/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -184,9 +170,131 @@
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="inondation">
 		<item-type>Number</item-type>
-		<label>Alerte</label>
+		<label>Inondation</label>
+		<description>Niveau d'alerte aux inondations</description>
+		<category>oh:meteoalerte:inondation</category>
+		<state readOnly="true">
+			<options>
+				<option value="0">Vert</option>
+				<option value="1">Jaune</option>
+				<option value="2">Orange</option>
+				<option value="3">Rouge</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="neige">
+		<item-type>Number</item-type>
+		<label>Neige</label>
+		<description>Niveau d'alerte aux évènements neigeux</description>
+		<category>oh:meteoalerte:neige</category>
+		<state readOnly="true">
+			<options>
+				<option value="0">Vert</option>
+				<option value="1">Jaune</option>
+				<option value="2">Orange</option>
+				<option value="3">Rouge</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="canicule">
+		<item-type>Number</item-type>
+		<label>Canicule</label>
+		<description>Niveau d'alerte aux fortes températures</description>
+		<category>oh:meteoalerte:canicule</category>
+		<state readOnly="true">
+			<options>
+				<option value="0">Vert</option>
+				<option value="1">Jaune</option>
+				<option value="2">Orange</option>
+				<option value="3">Rouge</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="grand-froid">
+		<item-type>Number</item-type>
+		<label>Grand Froid</label>
+		<description>Niveau d'alerte aux températures négatives</description>
+		<category>oh:meteoalerte:grand_froid</category>
+		<state readOnly="true">
+			<options>
+				<option value="0">Vert</option>
+				<option value="1">Jaune</option>
+				<option value="2">Orange</option>
+				<option value="3">Rouge</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="avalanches">
+		<item-type>Number</item-type>
+		<label>Avalanches</label>
+		<description>Niveau d'alerte aux avalanches</description>
+		<category>oh:meteoalerte:avalanches</category>
+		<state readOnly="true">
+			<options>
+				<option value="0">Vert</option>
+				<option value="1">Jaune</option>
+				<option value="2">Orange</option>
+				<option value="3">Rouge</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="vague-submersion">
+		<item-type>Number</item-type>
+		<label>Vague Submersion</label>
+		<description>Niveau d'alerte aux vagues submersion</description>
+		<category>oh:meteoalerte:vague_submersion</category>
+		<state readOnly="true">
+			<options>
+				<option value="0">Vert</option>
+				<option value="1">Jaune</option>
+				<option value="2">Orange</option>
+				<option value="3">Rouge</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="pluie-inondation">
+		<item-type>Number</item-type>
+		<label>Pluie Inondation</label>
+		<description>Niveau d'alerte aux inondations causée par les précipitations</description>
```

Comment:
causées

Reply:
In fact, I am even not sure if it should rather be "cause" ;) 
Depends on if you link to niveau or innondations.

Reply:
Ho sh**t ! Done :)

Reply:
Not yet fixed

Reply:
Done !